src/connector/src/source/kafka/split.rs

```diff
@@ -24,6 +24,12 @@ pub struct KafkaSplit {
     pub(crate) partition: i32,
     pub(crate) start_offset: Option<i64>,
     pub(crate) stop_offset: Option<i64>,
+    #[serde(skip)]
+    /// Used by shared source to hackily seek to the latest offset without fetching start offset first.
+    /// XXX: But why do we fetch low watermark for latest start offset..?
```

Comment:
Does it mean that theoretically we can also follow this approach for the Kafka sources specified `scan.startup.mode='latest'` by users?

Reply:
I do think so. @tabVersion so why do we fetch low watermark instead of passing Offset::End?

Reply:
Note that pulsar uses `PulsarEnumeratorOffset` in `PulsarSplit`, so I think it's perfectly ok.